mkdocs.yml

```diff
@@ -111,6 +111,7 @@ nav:
       - AdnRibbonMuscle     : maya/ribbon.md
       - AdnSimshape         : maya/simshape.md
       - AdnEdgeEvaluator    : maya/edge_evaluator.md
+      - AdnGlue             : maya/glue.md
```

Comment:
The order here does not match the entry in the index page. Is that intentional?

Reply:
I had doubts about the right order. The order did not match before either. I personally don't like much the order that we currently have. I think we should add sub-sections and reorder everything. Anyways, now I have moved AdnGlue after Edge Evaluator in both places.